Viewer/src/viewer/viewer.ts

```diff
@@ -198,6 +198,7 @@ export abstract class AbstractViewer {
         this.onInitDoneObservable.add(() => {
             this._isInit = true;
             this.engine.runRenderLoop(this._render);
+            this._prepareVR();

```

Comment:
Is it something we want all the time?

Reply:
So we have to make sure not to ship it before we can control it as it will change the default behavior else

Reply:
agreed, I will continue in this branch